### PR TITLE
Ecosystems stub

### LINKED ISSUE #3: What are the requirements for a successful community-centered ecosystem?
- [ ] What are the strengths of these communities?
- [ ] What are the weaknesses of these communities?

### PR DESCRIPTION
**Pull request by steffansluis**
This closes #3.

### CODE REVIEW

**Comment by steffansluis** [comment]
@bartist @WouterKvG Please review

File: ecosystem/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Requirements for a successful community-centered ecosystem
+
+This section introduces the term Software Ecosystem and its relation to a community, as well as a model for quantifying its characteristics. It also lists several challenges in creating and maintaining a successful community.
+
+
+## Software Ecosystems
+
+A Software Ecosystem (SECO) is defined by {{"van2010software" | cite}} van den Berk, Jansen, & Luinenburg, 2010, to be: 'A set of actors functioning as a unit and interacting with a shared market for software and services, together with the relationships among them.' A SECO consists of a central hub, a platform and niche players. In context with the subject of this research report, the platform is the FeedbackFruits platform, the niche players are its users, mostly teachers and students, and the hub is the FeedbackFruits company. In this context, the problem description becomes: how can the hub stimulate niche creation in the SECO?
```
Comment (WouterKvG) [inline ecosystem/README.md]: I would put the reference at the end of the sentence. Also, the ' should be before the .

File: ecosystem/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+According to {{"van2010software" | cite}}, niche creation can be stimulated by determining the desired future state of the SECO. To do this, both the current state and the desired improvements need to be known. In order for the hub to gain insight about current state of the ecosystem, the SECO Software Assesment Model can be used. It defines a broad spectrum of characteristics of the SECO that can be made quantifiable.
```
Comment (WouterKvG) [inline ecosystem/README.md]: - > According to [1]
  
  does not read very well. I would change it to something like "According to van den Berg et al."
- > gain insight about __the__ current state

File: ecosystem/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Software Ecosystems
+
+A Software Ecosystem (SECO) is defined by van den Berk, Jansen, & Luinenburg, 2010 {{"van2010software" | cite}} to be: 'A set of actors functioning as a unit and interacting with a shared market for software and services, together with the relationships among them.' A SECO consists of a central hub, a platform and niche players. In context with the subject of this research report, the platform is the FeedbackFruits platform, the niche players are its users, mostly teachers and students, and the hub is the FeedbackFruits company. In this context, the problem description becomes: how can the hub stimulate niche creation in the SECO?
+
+Niche creation can be stimulated by determining the desired future state of the SECO {{"van2010software" | cite}}. To do this, both the current state and the desired improvements need to be known. In order for the hub to gain insight about current state of the ecosystem, the SECO Software Assesment Model can be used. It defines a broad spectrum of characteristics of the SECO that can be made quantifiable.
```
Comment (WouterKvG) [inline ecosystem/README.md]: `gain insight about`**the**`current state`

File: ecosystem/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Niche creation can be stimulated by determining the desired future state of the SECO {{"van2010software" | cite}}. To do this, both the current state and the desired improvements need to be known. In order for the hub to gain insight about the current state of the ecosystem, the SECO Software Assesment Model can be used. It defines a broad spectrum of characteristics of the SECO that can be made quantifiable.
+
+To gain insights about the desired improvements, some kind of knowledge management will be needed in order to aggregate the information from all of the niche players. However, due to the use of personalized learning methodologies in the FeedbackFruits platform and the broad spectrum of subjects covered by niche players, a lot of this information is tacit knowledge. In {{"mcdermott2000knowing" | cite}}, McDermott, 2000, states that sharing tacit knowledge requires interaction and proceeds to provide a list of critical factors for building a community that can share such knowledge.
```
Comment (WouterKvG) [inline ecosystem/README.md]: > In {{"mcdermott2000knowing" | cite}}, McDermott, 2000,

Zou ik van maken `In McDermott (2002)`. En dan de referentie weer naar het einde van de zin. Dan klopt de zin alleen niet meer. Dus misschien `In _title_, McDermott (2002) states that..`

File: ecosystem/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Communities of practice
+
+Communities of practice enable person-to-person interaction while still engaging a whole group in advancing their field. As such, they are ideal vehicles for sharing tacit knowledge.
+In {{"mcdermott2000knowing" | cite}}, McDermott, 2000, list 10 critical success factors in building communities of practice. He separates these factors into four challenges:
```
Comment (WouterKvG) [inline ecosystem/README.md]: Zelfde als hierboven

**Comment by WouterKvG** [comment]
Looks good. Though it feels like some sort of conclusion is missing. But that should perhaps follow in another section.

**Comment by steffansluis** [comment]
@WouterKvG Yes, I had that feeling too. I think the conclusion should be in another section, but it still feels incomplete. I don't really know how to continue it though...

**Comment by steffansluis** [comment]
@WouterKvG @bartist Your feedback has been processed! Please feel free to click the merge button if you are satisfied with this PR.

**Pull request merged**